compiler/test/grainfmt/data_docs.expected.gr

```diff
@@ -15,6 +15,7 @@ and type Event = (EventType, Token, TokenizeContext)
  *   Tokens can “contain” other tokens, even though they are stored in a flat
  *   list, through `enter`ing before them, and `exit`ing after them.
  */
+
```

Comment:
Why was this blankline added?

Reply:
It was always supposed to be here—the source has a blank line here, but the old formatter didn't preserve it.